Assignment10.rb

```diff
@@ -0,0 +1,26 @@
+#Write an example of how you can use super method with and without arguments.
+class Animal
+  def initialize(name)
+    @name = name
+  end
+    
+  def speak
+    puts "#{@name} makes a sound"
+  end
+end
+  
+class Dog < Animal
+  def initialize(name, breed)
+    super(name)  
+    @breed = breed
+  end
+    
+  def speak
+    super() 
```

Comment:
we dont need to use any parenthesis if there is no arguments.
eg: can use `super` instead of `super()`